composer.json

```diff
@@ -68,7 +68,7 @@
     "mockery/mockery": "~0.9",
     "phpunit/phpcov": "2.*",
     "bossa/phpspec2-expect": "~1.0",
-    "elife/api": "dev-master",
+    "elife/api": "dev-job_advert_api",
```

Comment:
this should not be necessary anymore, but `composer update elife/api` will probably need to run to pick up the new api from `dev-master`